src/queue.hpp

```diff
@@ -957,6 +967,7 @@ struct cvk_command_buffer_image_copy final : public cvk_command_batchable {
     size_t m_offset;
     std::array<size_t, 3> m_origin;
     std::array<size_t, 3> m_region;
+    cl_command_type m_copy_type;
```

Comment:
Reusing `cl_command_type` is a bit confusing but it'll do for a first implementation.